ion-hash/src/type_qualifier.rs

```diff
@@ -76,8 +76,8 @@ impl TypeQualifier {
             /*IonType::Clob => Clob,
             IonType::Blob => Blob,
             IonType::List => List,
-            IonType::SExpression => SExpression,
-            IonType::Struct => Struct,*/
+            IonType::SExpression => SExpression,*/
+            IonType::Struct => combine(Struct, 0),
```

Comment:
Shouldn't this have a `Some`/`None` type of check too for the `null` case (is this tested or am I missing something)?  Also to be consistent with the other code `0x00` as the literal is probably warranted.

Reply:
I meant to back this out.